advanced/lighttpd.conf.debian

```diff
@@ -59,7 +59,10 @@ url.rewrite = ( "^(?!/admin/).*\.js$"  => "pihole/index.js"   )
 # If the URL does not start with /admin, then it is a query for an ad domain
 $HTTP["url"] =~ "^(?!/admin)/.*" {
     # Create a response header for debugging using curl -I
-    setenv.add-response-header = ( "X-Pi-hole" => "A black hole for Internet advertisements." )
+    setenv.add-response-header = (
+        "X-Pi-hole" => "A black hole for Internet advertisements.",
+        "X-Frame-Options" => "DENY"
```

Comment:
Since all ads are going through here, you might want to just add a regex for `pihole/ ` separately.